src/combat.cpp

```diff
@@ -774,14 +776,14 @@ void Combat::doCombatHealth(Creature* caster, Creature* target, CombatDamage& da
 	}
 
 	if (canCombat) {
+		if (caster && params.distanceEffect != CONST_ANI_NONE) {
+			addDistanceEffect(caster, caster->getPosition(), target->getPosition(), params.distanceEffect);
+		}
+		
```

Comment:
I was showing this feature to a friend at work who does not know how to use GitHub. I include a review and I took the opportunity to show that it is possible to add comments in different parts of the same review. Sorry for any inconvenience @ranisalt